klaviyo-react-native-sdk.podspec

```diff
@@ -13,6 +13,7 @@ Pod::Spec.new do |s|
   s.platforms    = { :ios => "13.0" }
   s.source       = { :git => "https://github.com/klaviyo/klaviyo-react-native-sdk.git", :tag => "#{s.version}" }
   s.source_files = "ios/**/*.{h,m,mm,swift}"
+  s.resources    = ["ios/klaviyo-react-native-sdk-configuration.plist"]
```

Comment:
There is no spm equivalent of this or that's not needed right?

Reply:
Hmm, didn't think about SPM. I think since we are including `KlaviyoSwift` SDK in the podspec and also since RN only supports cocoapods we should be good. Also, there is no package file in RN, there is no way to add this plist as a resource.